presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -461,6 +465,23 @@ class SystemConfig : public ConfigBase {
   int32_t oldTaskCleanUpMs() const;
 };
 
+/// Provides access to connector properties defined in
+/// catalog/$connector_name.properties file.
+class ConnectorConfig : public ConfigBase {
+ public:
+  static constexpr std::string_view kConnectorName{"connector.name"};
+  static constexpr std::string_view kHiveImmutablePartitions{
```

Comment:
We have "cache.enabled=true" in the config file, should we add it now too?

Reply:
I don't think "cache.enabled=true" is used anymore, I think they're replaced by https://github.com/prestodb/presto/blob/5b604194c8908aeb0244771536a786351e81a991/presto-native-execution/presto_cpp/main/common/Configs.h#L196-L206

Reply:
Then maybe we should remove it from the config file.

Reply:
Yes, already did